### PR TITLE
Change Theme Button visible now in md screen

### PR DESCRIPTION
**Pull request by mohdhaider07**
All the changes made is navbar.js
and theme change button is removed from index.js file(home screen)

### CODE REVIEW

File: components/Navbar.js [hunk 1]
```diff
@@ -6,146 +6,180 @@ import MoreMenu from "./MoreMenu";
...
-      <nav className="flex items-center transition-colors flex-wrap bg-white drop-shadow-xl p-1 justify-center mb-10 dark:bg-black dark:text-white dark:transition-colors sticky top-0 z-10">
-        <Link href="/">
-          <a className="inline-flex items-center p-2 mr-4 ">
-            <span className="text-xl text-orange font-bold uppercase tracking-wide  ml-8">
-              {/* <img
+	return (
+		<>
+			<nav className="flex items-center flex-wrap transition-colors  bg-white drop-shadow-xl p-1 justify-center mb-10 dark:bg-black dark:text-white dark:transition-colors sticky top-0 z-10">
+				<Link href="/">
+					<a className="inline-flex items-center p-2 mr-4 ">
```
Comment (Suvraneel) [inline components/Navbar.js:35]: @mohdhaider07 
Please do not change lines that have not been changed.
This repository uses an indentation of 2 spaces in contrast to your formatter which uses 4 spaces.

**Review by Suvraneel — CHANGES_REQUESTED** [review]

File: components/Navbar.js (hunk 1 above)
Comment (mohdhaider07) [reply components/Navbar.js:35]: Okay

**Comment by Suvraneel** [comment]
@mohdhaider07  No difference bro.

Basically these lines shouldn't appear in diff:
image

Make a new PR if you need

**Comment by mohdhaider07** [comment]
>
I've made changes specifically in Navbar.js and Index.js, and I don't believe it's a significant issue. You can go ahead and merge them."

**Comment by Suvraneel** [comment]
@mohdhaider07  
I totally understand and agree with you. Indentation sizes don't matter.🖤

But in large codebases it's always advisable to preserve git history because it's easier to `git blame` someone incase something goes wrong. 

Moreover, modifying irrelevant lines also gives a false weightage to the extent of the PR (if I assign a level 1 to this PR, someone might disagree if he feels it was unjustified).

( I've no issues merging this, but we're supposed to be preaching good open-source practices. Hence, here we go )

**Comment by mohdhaider07** [comment]
@Suvraneel thanks for explaining got your point

**Pull request closed without merging**